git_machete/utils.py

```diff
@@ -295,3 +296,12 @@ def format_choice(c: str) -> str:
         else:
             return colored(c, AnsiEscapeCodes.ORANGE)
     return f" ({', '.join(map_truthy_only(format_choice, choices))}) "
+
+
+class Color(Enum):
```

Comment:
Hmmm this still doesn't abstract away ANSI escape codes... it's just a 1:1 wrapper over such codes 🤔

Reply:
What I'd like to see would rather be an enum that can only hold values that represent **edge colors** (so `GREEN`, `RED`, `GRAY`, `YELLOW`)... that will be translated to ANSI escape codes only when needed.

The enum itself should know nothing of ANSI escape codes.